src/components/Radio/Radio.mdx

```diff
@@ -0,0 +1,72 @@
+---
+name: Radio
+menu: Components
+route: /radio
+---
+
+import { Playground, Props } from 'docz'
+import { Radio, Loader } from '../../index.ts'
+
+# Radio
+
+## Examples
+
+### Basic usage
+
+<Playground>
+  <Radio.Group name="classes1" defaultValue="react">
```

Comment:
Yes! I do this because otherwise multiple RadioGroups will have the same name value.

This doesn't cause errors. It just won't give the wanted result. However, this should maybe be specified in the documentation.